source/_patterns/00-atoms/date/date.twig

```diff
@@ -0,0 +1 @@
+<time datetime="{{ date|date("Y-m-d") }}">{{ date|localizeddate('long', 'none') }}</time>
```

Comment:
The `datetime` attribute needs to be added the extension fashion, like https://github.com/libero/pattern-library/blob/121c4d37b53ba91398b645d1499d99eba1895609/source/_patterns/00-atoms/block/paragraph/paragraph.twig#L1-L3

Reply:
Should move this pattern into the `inline` folder, and make use of the `spaceless` filter to avoid accidental whitespace being added. So it would look like https://github.com/libero/pattern-library/blob/121c4d37b53ba91398b645d1499d99eba1895609/source/_patterns/00-atoms/inline/bold/bold.twig#L1-L9 with the addition of the `datetime` being set.